src/readers/morphologyHDF5.h

```diff
@@ -35,9 +121,19 @@ class MorphologyHDF5
                unsigned int expectedDimension,
                T& data);
 
+    template <typename T>
+    void _read(const std::string& group,
+               const std::string& _dataset,
+               const std::vector<size_t>& expectedDimensions,
+               T& data);
+
     HighFive::Group _group;
     Property::Properties _properties;
     std::string _uri;
+
+    int _file_version;
```

Comment:
I take it we're making the assumption that all morphologies within a container have the same version; this makes sense, but perhaps we should write that down somewhere (I assume I've missed it :()

Reply:
This is the version of the container, not of individual morphologies. "merged" is 3 and "unified" is 4. One can mix morphologies of different versions, e.g. v1.x, which is checked in the unit-tests, both `merged.h5` and `unified.h5` include `"simple.h5", "mitochondria.h5", "endoplasmatic-reticulum.h5"`. I'll need to add `simple-dendritric-spine.h5`.